.changeset/pink-pillows-impress.md

```diff
@@ -0,0 +1,6 @@
+---
+"@aws-amplify/ui-react-liveness": patch
+"@aws-amplify/ui-react": patch
```

Comment:
This was autogenerated, is this supposed to be here? We made no changes to ui-react